_posts/2018-02-15-IWD-2018-announcement.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  International Women's Day 2018! 
+date: 2018-02-20
+categories: wtmberlin iwd
+---
+
+We are happy to announce that we are joining the [Women Techmakers](https://www.womentechmakers.com/iwd18) Global Event Series Celebrating International Women's Day!
+
+The International Women's Day by Women Techmakers Berlin celebration is happening on **Thursday, March 22** and we would love for **you** to be part of it!
+
+Our location this year is the [Digital:Lab Volkswagen](https://goo.gl/maps/6haLVpAjUio).
+
+##### International Women's Day
+[International Women's Day (IWD)](https://www.internationalwomensday.com/) is marked on March 8th every year. In different regions, the focus of celebrations ranges from general respect and appreciation for women to a celebration of women's economic, political, and social achievements.
+
+> _Source:_ [Women Techmarkers](https://www.womentechmakers.com/iwd18)
+
+##### Building a New Horizon
+This year's theme is _Building a New Horizon_. 
+
+Building a new horizon means *creating* a new path, *opening* our perspective to new possibilities, asking *new* questions, *discovering* new worlds.
+
+This new horizon is built **together**, its elements are everyone's dreams, opinions and perspectives, to make better decisions in our companies, in our homes and families, to make our world, the only home we have ever known, a better place. 
+
+Join us in our journey to a tech community more diverse and inclusive!
+
+##### I want to join!
+Yay! RSVP at the [Women Techmakers Meetup event](https://www.meetup.com/Women-Techmakers-Berlin/events/247964417/).
+
+We follow the [Google Event Community Guidelines and Anti-Harassment Policy](https://www.google.com/events/policy/anti-harassmentpolicy.html) and the [Berlin Code of Conduct](http://berlincodeofconduct.org/).
+
+Also, if you are looking for inspiration, take a look at [last year's event](http://wtmberlin.com/blog/IWD-2017).
+
+Thank you and hope to see you at the event! :)
+
+<dl>Ely Flores, on behalf of the Women Techmakers Berlin Community.</dl>
+
+***
+###### About Women Techmarkers Berlin
+At Women Techmakers Berlin we are encouraging diversity in tech in Berlin since 2015. Get visibility, networking and resources to make the tech community of Berlin diverse and inclusive!
+
+Our meetup group is always open and inclusive to people of all backgrounds, regardless of gender, age, race, expertise, etc. Everyone is welcome.
+
+Join us at WTM Berlin [Facebook group](https://www.facebook.com/groups/1106085959408694) and follow us on [Facebook](https://www.facebook.com/WomenTechmakersBerlinPage/) and [Twitter](https://twitter.com/wtm_berlin).
+
+<img src="https://raw.githubusercontent.com/WTMBerlin/slides/master/logos/wtm_berlin_logo.png" width="70%" height="70%">
```

Comment:
@Pisush Thank you to the hint to that image, I like it more! :) I used the link that you mentioned directly. However, if you think it would be best to copy the image into this repo and use is it as a local file, we can update! 
Thanks!